changelog/v2.md

```diff
@@ -55,11 +50,11 @@ module.exports = {
 };
 ```
 
-Then update `public/plugins.json` and update the port on `src` from `8085` to `3000`. Finally run `npm install` to install/update the new dependencies.
+Then update `public/plugins.json` and update the port on `src` from `8085` to `3000`. Finally, run `npm install` to install the new dependencies.
 
 A `.env` file was initially included to prevent the browser from opening Flex on `npm start`. This is no longer needed and can be removed.
 
-You also need to move your jest configuration into `craco.config.js`, otherwise CRA will ignore them.
+If you have added any jest configuration, you must move these into `craco.config.js` so that `create-react-app` will not ignore them.
```

Comment:
nit: `If you have added a custom jest configuration, you must move it into...`

Reply:
This is a rebase with master; we need to update master and update this branch later.